src/esbuild_plugins/islands.ts

```diff
@@ -46,6 +63,28 @@ export const island_wrapper = (mode: "ssr" | "dom", dir: string): Plugin => ({
 				filename,
 			});
 
+			if (island && mode === "dom") {
+				const name = island[1];
+				code += `
+					const load = performance.now();
+					const targets = document.querySelectorAll("one-claw[name='${name}']");
```

Comment:
Note that since #31 we only target top-level islands, so this should probably be:

```ts
const targets = document.querySelectorAll("one-claw[name='xxx']:not(one-claw onew-claw)");
```